src/pages/Search/DesktopResults.tsx

```diff
@@ -39,28 +39,26 @@ function DesktopResults({ results }: { results: CareProviderSearchResult[] }) {
           key="desktop-map"
           className="position-sticky top-0"
         >
-          <div className="border-right border-left border-base-lighter">
```

Comment:
unrelated change here and in MobileResults - just noticed that we still had a border, which we only needed b/c of the stamen tiles that had a white bg